source/_components/here_travel_time.markdown

```diff
@@ -0,0 +1,145 @@
+
+## {% linkable_title Setup %}
+
+You need to register for an API key by following the instructions [here](https://developer.here.com/documentation/routing/topics/introduction.html?create=Freemium-Basic&keepState=true&step=account).
+
+HERE offers a Freemium Plan which includes 250.000 free Transactions per month. For the Routing API one transaction equals one request with one starting point (no multistop). More information can be found [here](https://developer.here.com/faqs#payment-subscription)
+
+By default HERE will deactivate your account if you exceed the free Transaction limit for the month. You can add payment details to reenable your account as described [here](https://developer.here.com/faqs)
+
+## {% linkable_title Configuration %}
+
+To enable the sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example entry for configuration.yaml
+sensor:
+  - platform: here_travel_time
+    app_id: "YOUR_APP_ID"
+    app_code: "YOUR_APP_CODE"
+    origin: "51.222975,9.267577"
+    destination: "51.257430,9.335892"
+    name: "HERE Travel Time Home -> Work"
+    travel_mode: "car"
+    route_mode: "shortest"
+    traffic_mode: "true"
+```
+
+{% configuration %}
+app_id:
+  description: "Your application's API id (get one by following the instructions above)."
+  required: true
+  type: string
+app_code:
+  description: "Your application's API code (get one by following the instructions above)."
+  required: true
+  type: string
+origin:
+  description: "The starting point for calculating travel distance and time."
+  required: true
+  type: string
+destination:
+  description: "One or more locations to use as the finishing point for calculating travel distance and time."
+  required: true
+  type: string
+name:
+  description: A name to display on the sensor. The default is "HERE Travel Time".
+  required: false
+  type: string
+travel_mode:
+  description: "You can choose between: `car`, `pedestrian`, `publicTransport` or `truck`. The default is `car`."
+  required: false
+  type: string
+route_mode:
+  description: "You can choose between: `fastest`, or `shortest`. This will determine whether the route is optimized to be the shortest and completely disregard traffic and speed limits or the fastest route according to the current traffic information. The default is `fastest`"
+  required: false
+  type: string
+traffic_mode:
+  description: "You can choose between: `true`, or `false`. Decide whether you want to take the current traffic condition into account. Default is `false`."
+  required: false
+  type: string
+{% endconfiguration %}
+
+## {% linkable_title Dynamic Configuration %}
+
+Tracking can be setup to track entities of type `device_tracker`, `zone`, `sensor` and `person`. If an entity is placed in the origin or destination then every 5 minutes when the platform updates it will use the latest location of that entity.
```

Comment:
```suggestion
Tracking can be set up to track entities of type `device_tracker`, `zone`, `sensor` and `person`. If an entity is placed in the origin or destination then every 5 minutes when the platform updates it will use the latest location of that entity.
```